README.md

```diff
@@ -41,9 +42,21 @@ You will only need to run this command when dependencies change in `package.json
 
 We use npm scripts and [Webpack][] as our build system.
 
+### Microservices Overview
```

Comment:
Could you maybe use this instead

> Podium is built in a microservices architecture which works in following way:
> 
> * Podium Gateway, serves the Angular application and also handles web traffic.
> * Podium UAA, is a User Account and Authentication server based on Spring Security. This server provides OAuth2 tokens for securing the gateway.
> * Podium-Registry, a runtime application on which all applications registers and get their configuration from. It also provides runtime monitoring dashboards.
>

Reply:
I've added it but I feel we should still tell people the order in which to run this.

Reply:
Then remove this part:

>  There are two services in this repository, the uaa service and the gateway service. There is a third the Podium Registry that has its own repository and needs to be running for the others to work.

Reply:
it is gone now